src/shogun2-core/src/main/java/de/terrestris/shogun2/model/layer/source/TileWmsLayerDataSource.java

```diff
@@ -37,8 +37,7 @@
 	 * Whether to request the layer with TILED=true.
 	 */
 	@Column(name = "REQUEST_WITH_TILED")
-	@ColumnDefault(value = "true")
-	private Boolean requestWithTiled = true;
+	private Boolean requestWithTiled = Boolean.TRUE;
```

Comment:
Is there a reason for this change? I think this will lead to different behaviour:

Using `@ColumnDefault` will apply the `DEFAULT` value to the associated column in the DDL, i.e. if you use the hibernate ddl UPDATE option on a database that is missing this column, the values in the database will initially be "true" afterwards.

With your change, these values would be null in case of UPDATE, right?

Reply:
The `value` provided in `@ColumnDefault` annotation as column default is vendor specific since it will be applied to DDL. In case of Oracle databases it has to be a numerical value (1 in this case), here. Using `"true"` loads to the following (errornous) DDL:

```
create table tilewmslayerdatasources (request_with_tiled number(1,0) default true, 
    id number(10,0) not null, tilegrid_id number(10,0), primary key (id))
```

If we use a non-nullable column here (e.g. `@Column(name = "REQUEST_WITH_TILED", nullable = false)` ) these values would not be null on hibernate update, right?

Reply:
Ok. I understand. I don't think that this is a major issue, so in my eyes you can leave it as proposed